File: Practice/prime.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int num,c=0,r;
    cout<<"Enter a number: ";
    cin>>num;
    for (int i = 1; i <= num; i++)
    {
        if (num % i == 0)
        {
            c++;
        }
        
    }
    if (c == 2)
    {
        cout<<"The number entered is prime";
    }
    

}

File: java/May-7/prime.java
class prime
{
     public static void main(String[] args) {

          for(int i=15;i<=55;i++)
          {
               int cnt=0;
               for(int j=1;j<=i;j++)
               {
                    if(i%j==0)
                    {
                         cnt++;
                    }
               }
               if(cnt == 2)
               {
                    System.out.println(i);
               }

          }
     }
}
